.circleci/config.yml

```diff
@@ -45,6 +45,8 @@ jobs:
     parameters:
       go_version:
         type: string
+    environment:
+      CI_E2E_FILENAME: "net_done"
```

Comment:
I don't know if this is what we want.

> PRs should use an artifact that is pinned according to the current submodule.
The pinned submodule can be overridden to allow pinning to a specific feature branch.

Correct me if I'm wrong...

The most recent changes to the go-algorand CI mean that we are only currently uploading e2e test files for builds of the `rel-nightly` branch unless someone manually runs the CI w/ the parameter supplied.
See https://github.com/algorand/go-algorand/blob/master/.circleci/config.yml#L635 and https://github.com/algorand/go-algorand/blob/master/test/scripts/e2e.sh#L166

This means that `net_done.tar.bz2`, which is now the default name when `CI_E2E_FILENAME` is not found https://github.com/algorand/go-algorand/blob/master/test/scripts/e2e.sh#L180 should not get updated almost ever.

We need to modify the go-algorand CI to upload files with their commit hash associated w/ them so that we can look at our submodule and pull the e2e test files that were generated from the same commit that our submodule is running.

Reply:
this makes sense. I'll make the changes.

Reply:
in go-algorand, do we want to create an artifact for every merge to master or create another artifact with commit hash as part of rel/nightly build?

Reply:
or can we tag the rel/nightly artifact with commit hash?

Reply:
or can we tag the rel/night artifact by adding the commit hash in the metadata?

Reply:
I think you can select a specific artifact by providing a longer "prefix" in order to make it unique / searchable. For example: `faffce2f/rel-nightly`